crates/red_knot_module_resolver/src/typeshed/versions.rs

```diff
@@ -80,7 +81,7 @@ impl fmt::Display for TypeshedVersionsParseErrorKind {
     }
 }
 
-#[derive(Debug, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq, Clone)]
 pub struct TypeshedVersions(FxHashMap<ModuleName, PyVersionRange>);
```

Comment:
What's the reason for supporting `Clone`? Seems expensive

Reply:
It's required for snapshotting. I'll put the `FxHashMap` behind an `Arc` to make it less expensive to clone. It makes sense for the inner mapping to be immutable: if the user changes the `VERSIONS` file, we'll need to reparse the whole thing anyway.